tests/config_mgmt_test.py

```diff
@@ -78,6 +78,30 @@ def test_break_out(self):
         self.dpb_port4_4x25G_2x50G_f_l(curConfig)
         return
 
+    def test_extra_brk_cfg_tables_cases(self):
```

Comment:
We should have 3 different test cases:
1>  Tables that we should ignore specifically even with/without Yang, and no warning. (we probably don't need this based on my latest comments: https://github.com/Azure/sonic-buildimage/pull/6801#issuecomment-840045326 ), and I don't see other cases.
2>  Tables that does not have Yang models but not depending on port to be deleted. By default , it should continue with warning.
3>  Tables that does not have Yang models but depending on the port to be deleted. It should ask user to confirm.